README.md

```diff
@@ -1,11 +1,20 @@
+
 # **Distributed Deep Learning on AWS Using MXNet and TensorFlow**
 
 [AWS CloudFormation](https://aws.amazon.com/cloudformation), which creates and configures Amazon Web Services resources with a template, simplifies the process of setting up a distributed deep learning cluster. The AWS CloudFormation Deep Learning template uses the [Amazon Deep Learning AMI](https://aws.amazon.com/marketplace/pp/B01M0AXXQB) (which provides MXNet, TensorFlow, Caffe, Theano, Torch, and CNTK frameworks) to launch a cluster of [EC2](https://aws.amazon.com/ec2) instances and other AWS resources needed to perform distributed deep learning.  
-With this template, we continue with our mission to make [distributed deep learning easy](https://aws.amazon.com/blogs/compute/distributed-deep-learning-made-easy/). AWS CloudFormation creates all resources in the customer account. 
+With this template, we continue with our mission to make [distributed deep learning easy](https://aws.amazon.com/blogs/compute/distributed-deep-learning-made-easy/). AWS CloudFormation creates all resources in the customer account.
 
 ## What's New?  
 We've updated the AWS CloudFormation Deep Learning template to add some exciting new features and capabilities.
 
+### Nov 7 2018
+* Introduce AWS DLAMI Conda v16.0 - For developers who want pre-installed pip packages of deep learning frameworks in separate virtual environments, the Conda-based AMI is available in **[Ubuntu](https://aws.amazon.com/marketplace/pp/B077GCH38C),**  **[Amazon Linux](https://aws.amazon.com/marketplace/pp/B077GF11NF)** and **[Windows 2016](https://aws.amazon.com/marketplace/pp/B077Y5DJ8H?qid=1517601174147&sr=0-1&ref_=srh_res_product_title)** versions. Check [AWS DLAMI Official webpage](https://aws.amazon.com/machine-learning/amis/) for more details.
+* We now support 5 more AWS regions- sa-east-1, us-west-1, ca-central-1, eu-west-2, eu-west-3.
```

Comment:
can you rephrase as we support the following regions...

Reply:
> can you rephrase as we support the following regions...

done.